### PR TITLE
Add LibpcapVersionAttribute

### PR DESCRIPTION
**Pull request by kayoub5**
Usage
```cs
        [Test]
        [LibpcapVersion(">2.0.0")]
        public void LibpcapVersionAttributeTest()
        {
            // Test will only run if libpcap version > 2.0.0
            Assert.Fail("Libpcap coming from the future");
        }
```
See https://docs.npmjs.com/cli/v6/using-npm/semver for examples on ranges

### CODE REVIEW

**Comment by kayoub5** [comment]
@chmorgan waiting for your review

**Review by chmorgan — CHANGES_REQUESTED** [review]
How did you figure it out? It seems pretty obscure.

Are there any tests where we could put this into place via another commit before merging it? As policy we should make use of code we add to see how it looks/works.

**Comment by kayoub5** [comment]
@chmorgan Tests updated

**Review by chmorgan — APPROVED** [review]

**Pull request merged**
